typescript/infra/scripts/utils.ts

```diff
@@ -59,6 +60,13 @@ export async function getEnvironment() {
   return assertEnvironment(await getEnvironmentFromArgs());
 }
 
+type SdkEnvironment = keyof typeof coreEnvironments;
+export const DeployEnvToSdkEnv: Record<DeployEnvironment, SdkEnvironment> = {
```

Comment:
Two nits but nbd:
1. Convention would be camelCase I think (`deployEnv...`)
2. scripts/utils seems like an odd place to keep consts like this

Reply:
Some `consts` file/folder would be my first thought, but this isn't the only instance of const-y things in infra utils files so ship may have sailed on this :)

Reply:
I'll punt this to the next person then :)